ci-operator/step-registry/ipi/conf/vsphere/permission/ipi-conf-vsphere-permission-ref.yaml

```diff
@@ -0,0 +1,14 @@
+ref:
+  as: ipi-conf-vsphere-permission
+  from: upi-installer
+  commands: ipi-conf-vsphere-permission-commands.sh
+  resources:
+    requests:
+      cpu: 10m
+      memory: 100Mi
+  credentials:
+  - namespace: test-credentials
+    name: devqe-secrets
```

Comment:
This is newly added in vault?

Reply:
It exists since we added devqe vsphere and only added mininal permission user to it for this PR

Reply:
suggest to rename `vcenter_username_permission` to `vcenter_username_minimal_permission`

Reply:
Updated